portal/guides/mint-specific-nft-with-nextjs.mdx

```diff
@@ -0,0 +1,522 @@
+    3. Mint NFT
+
+## 1. Set up
+
+The easiest way to set up a Next.js app is to use the template which can be [found here.](https://github.com/thirdweb-example/next-typescript-starter) You can “Use this template” and create your own repo, which can be cloned and opened in your code editor. 
+
+Install the required dependencies:
+
+```bash
+npm install
+# or
+yarn install
+```
+
+Then, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+```
+
+Open [http://localhost:3000](http://localhost:3000/) with your browser to see the result. At the moment, all that your app allows is for you to connect your wallet.
+
+For this project, we will use Chakra to style our components. You can install these dependencies: 
+
+```bash
+npm i @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
+# or
+yarn add @chakra-ui/react @emotion/react@^11 @emotion/styled@^11 framer-motion@^6
+```
+
+Don’t forget to add the `ChakraProvider`, afterwhich your `_app.tsx` should look like this:
+
+```tsx
+import type { AppProps } from 'next/app';
+import { ChainId, ThirdwebProvider } from '@thirdweb-dev/react';
+import { ChakraProvider } from '@chakra-ui/react';
+
+// This is the chainId your dApp will work on.
+const activeChainId = ChainId.Rinkeby;
+
+function MyApp({ Component, pageProps }: AppProps) {
+  return (
+    <ThirdwebProvider desiredChainId={activeChainId}>
+      <ChakraProvider>
+        <Component {...pageProps} />
+      </ChakraProvider>
+    </ThirdwebProvider>
+  );
+}
+
+export default MyApp;
+```
+
+**NB:** If you open `_app.tsx` (in your `pages` folder), you’ll see that you can set the chain on which you want your app to work. In this tutorial, we are working on the Rinkeby testnet so we have changed it from this: 
+
+```tsx
+// This is the chainId your dApp will work on.
+const activeChainId = ChainId.Mainnet;
+```
+
+to this:
+
+```tsx
+// This is the chainId your dApp will work on.
+const activeChainId = ChainId.Rinkeby;
+```
+
+**Awesome🔥  Let’s create some NFTs**
+
+## 2. Display Un-Minted NFTs
+
+### Add Wallet Connect Button
+
+We can use the thirdweb `useAddress` and `useMetamask` to easily create a connect button. Here is what your `pages/index.tsx` should look like:
+
+```tsx
+import { useAddress, useMetamask } from '@thirdweb-dev/react';
+import type { NextPage } from 'next';
+
+const Home: NextPage = () => {
+  const address = useAddress();
+  const connectWithMetamask = useMetamask();
+
+  return (
+    <div>
+      {address ? (
+        <h1>Connected!</h1>
+      ) : (
+        <button onClick={connectWithMetamask}>Connect Metamask Wallet</button>
+      )}
+    </div>
+  );
+};
+
+export default Home;
+```
+
+### Create nftCard component
+
+We will display each of our NFTs using an `nftCard` component.  Chakra’s `SimpleGrid`, `Flex` , `Box` and `Image` components will help us build each NFT card. You can create a new file called `nftCard.tsx` in a new folder called `components` .
```

Comment:
```suggestion
We will display each of our NFTs using an `nftCard` component.  Chakra’s `SimpleGrid`, `Flex` , `Box` and `Image` components will help us build each NFT card. You can create a new file called `NftCard.tsx` in a new folder called `components` .
```

NftCard should be PascalCase not camelCase, components should be capitalized in the file name. Update this throughout the guide and the example code